frontend/src/metabase/components/AddToDashSelectDashModal.jsx

```diff
@@ -58,10 +60,21 @@ export default class AddToDashSelectDashModal extends Component {
                     title="Add Question to Dashboard"
                     closeFn={this.props.closeFn}
                 >
+                <div className="flex flex-column">
```

Comment:
@wtfleming Try adding `align center` on this parent and I believe this should fix the alignment issue you mentioned.